pkg/util/kubernetes/apiserver/services.go

```diff
@@ -63,18 +63,25 @@ func (smb *ServiceMapperBundle) mapServices(nodeName string, pods v1.PodList, en
 	return nil
 }
 
-// GetPodSvcs is used when the API endpoint of the DCA to get the services of a pod is hit.
-func GetPodSvcs(nodeName string, podName string) []string {
-	smb, found := cache.Cache.Get(nodeName)
+// GetPodServiceNames is used when the API endpoint of the DCA to get the services of a pod is hit.
+func GetPodServiceNames(nodeName string, podName string) []string {
+	var serviceList []string
+	smbInterface, found := cache.Cache.Get(nodeName)
```

Comment:
gocache keys should be more explicit and unique. `cache.BuildAgentKey()` offers a common way to build them